### PR TITLE
Preferences cleanup C (ExporterFactory & setPreviewStyle)

### PR DESCRIPTION
**Pull request by calixtus**
Follow-up to #9866

In preparation for refactoring the getCustomExporters to the new preferences pattern, I had to move the JournalAbbreviationRepository argument in the constructor to the actual export method. As the journalAbbreviationRepository variable in Globals is overwritten when the preferences are imported or reset, this might have also fixed some hidden bugs. As for now I leave the call to Globals in some places, as fixing that call too would explode this PR.

Commits are mostly self explaining.
Refactoring was made in several iterations, so I had to introduce the JournalAbbreviationRepository in some steps, just to remove it in the next one.

Also made some very small fixes on the fly.

### Mandatory checks
- [ ] Change in `CHANGELOG.md` described in a way that is understandable for the average user (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked developer's documentation](https://devdocs.jabref.org/): Is the information available and up to date? If not, I outlined it in this pull request.
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not, I created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, I submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java [hunk 1]
```diff
@@ -56,50 +57,62 @@
...
                 .setOnDragOver(this::handleOnDragOver)
-                .withValidation(LinkedFileViewModel::fileExistsValidationStatus);
-
-        listView.setCellFactory(cellFactory);
+                .withValidation(LinkedFileViewModel::fileExistsValidationStatus)
+                .install(listView);
         listView.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
 
-        decoratedModelList = new UiThreadObservableList<>(viewModel.filesProperty());
-        Bindings.bindContentBidirectional(listView.itemsProperty().get(), decoratedModelList);
```
Comment (Siedlerchr) [inline src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java:102]: Why was this removed?

Comment (calixtus) [reply src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java:102]: Its not, it stayed in the constructor, while other initialization stuff is moved to the initialize method.

Comment (Siedlerchr) [reply src/main/java/org/jabref/gui/fieldeditors/LinkedFilesEditor.java:102]: ah I see!  Github diff was a bit weird at that point

**Review by Siedlerchr — APPROVED** [review]

**Review by koppor — APPROVED** [review]

**Comment by calixtus** [comment]
Yay
